popup/popup.js

```diff
@@ -343,6 +344,10 @@ function createStyleElement(style) {
   return entry;
 }
 
+function capitalize(s) {
+  return s[0].toUpperCase() + s.slice(1);
+}
+
 function styleExcluded({exclusions}, type) {
```

Comment:
I don't think we should manipulate translations like this... even using [`text-transform`](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#CSS_issues) isn't reliable.

Reply:
It doesn't manipulate translations. It is just a helper function to capitalize the first character. For example:
`'installPreferScheme' + 'dark'` -> `'installPreferScheme' + 'Dark'`

Reply:
Oh ok... I missed that because I was lacking caffeine 😸